src/styles/blocks/star.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &--4 :nth-child(-n + 4) {
```

Comment:
There is an extra space between '&--4' and ':nth-child(-n + 4)'. According to the code style rules, there should be no space between the element and the pseudo-class.